pkg/alerts/alertsHandler/alertsHandler.go

```diff
@@ -173,6 +181,115 @@ func ProcessSilenceAlertRequest(ctx *fasthttp.RequestCtx) {
 	utils.WriteJsonResponse(ctx, responseBody)
 }
 
+func ProcessTestContactPointRequest(ctx *fasthttp.RequestCtx) {
+	var testContactRequest TestContactPointRequest
+	if err := json.Unmarshal(ctx.PostBody(), &testContactRequest); err != nil {
+		utils.SendError(ctx, "Failed to unmarshal json", "ProcessTestContactPointRequest: Request Body: "+string(ctx.PostBody()), err)
```

Comment:
nit: you don't need `ProcessTestContactPointRequest:` (but keep the rest of that string). This `SendError()` function automatically adds the function name

Same comment many other places you're calling `SendError()`

Just for your own benefit, you might want to make `utils.SendError()` execute once and then look at the log file for what the output looks like